src/main/scala/uk/gov/homeoffice/drt/model/DepartureAirport.scala

```diff
@@ -0,0 +1,60 @@
+package uk.gov.homeoffice.drt.model
+
+case class DepartureAirport(country: String, airportCode: String, airportName: String)
+
+object DepartureAirport {
+
+  val athensCountryAirportList = List(
```

Comment:
I'm not sure what this name means. Is it the departure airports that the athens office is interested in? If so then maybe rename it something like `athensDeparturePorts`